data/human/hails.txt

```diff
@@ -1708,6 +1708,91 @@ phrase "friendly civilian"
 		", but I still had to take a big loan to buy my first ship."
 		", but it was my grandma's inheritance that allowed me to buy this ship."
 
+phrase "friendly civilian"
+	word
+		"Don't be too picky about the jobs you take. It's better to"
+	word
+		" "
+	word
+		" ship crooks, than having a crooked ship."
```

Comment:
The leading space in this last section is unnecessary given that the previous section adds a space already.

Reply:
Thanks, I removed the unnecessary spaces.